src/components/Chip/Chip.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC, ReactNode, useMemo } from 'react'
+import { VariantProps, tv } from 'tailwind-variants'
+
+export type Props = {
+  label: ReactNode
+  size?: VariantProps<typeof chip>['size']
+  className?: string
+}
+
+export const chip = tv({
+  base: [
+    'smarthr-ui-Chip',
+    'shr-rounded-full',
+    'shr-border',
+    'shr-border-solid',
+    'shr-border-default',
+    'shr-leading-none',
+    'contrast-more:shr-border-high-contrast',
+  ],
+  variants: {
+    size: {
+      S: ['shr-text-sm', 'shr-px-0.5', 'shr-py-0.25'],
```

Comment:
コンポーネント系のサイズ指定が現状小文字の `s` になってる物が多いため、一旦そっちに寄せて、どこかのタイミングで揃えたい、という気持ちです。

```suggestion
      s: ['shr-text-sm', 'shr-px-0.5', 'shr-py-0.25'],
```